mempool/clist_mempool.go

```diff
@@ -360,7 +366,12 @@ func (mem *CListMempool) removeTx(tx types.Tx, elem *clist.CElement, removeFromC
 //
 // The case where the app checks the tx for the second and subsequent times is
 // handled by the resCbRecheck callback.
-func (mem *CListMempool) resCbFirstTime(tx []byte, peerID uint16, res *abci.Response) {
+func (mem *CListMempool) resCbFirstTime(
+	tx []byte,
+	peerID uint16,
```

Comment:
Does it make sense to rename these two variables to `peerShortID` and `peerID` resp.?

Reply:
Yeah, the naming isn't ideal. I think we should keep the `peerID` name, since that's what it's called throughout the Mempool, but I changed `TxInfo.SenderAddress` and `peerAddress` to `PeerFullID`. What do you think?

Reply:
so peerID in mempool and p2p would mean different things... I think It's fine, but we should probably rename `PeerFullID` to `peerP2PID`, so it's obvious that `peerP2PID` equals to `p2p.ID`. Thoughts?

Reply:
Sure thing, done.